piv_error.py

```diff
@@ -52,8 +55,9 @@ def recursiveGaussSmooth(data, L):
 	b3 = 0.422205*(q**3)
```

Comment:
They are the filter coefficients for a Gaussian smoother.  They are defined in the paper i mention in the preamble of the function. The filter is supposed to be super fast, but I tested it against the scipy gaussian filter and the scipy one is much faster. I have changed that in the code, just haven't committed it yet.